app/code/Magento/Bundle/Block/Catalog/Product/View/Type/Bundle.php

```diff
@@ -178,6 +178,16 @@ public function getJsonConfig()
                 $configValue = $preConfiguredValues->getData('bundle_option/' . $optionId);
                 if ($configValue) {
                     $defaultValues[$optionId] = $configValue;
+
+                    // Overwrite default QTY (if set)
+                    $qtyValue = $preConfiguredValues->getData('bundle_option_qty/' . $optionId);
+                    if ($qtyValue) {
+                        $configValues = (is_array($configValue)) ? $configValue : [$configValue] ;
+                        $qtyValues = (is_array($qtyValue)) ? $qtyValue : [$qtyValue] ;
```

Comment:
- please remove the extra space before the `;` (2x)
- `(is_array($qtyValue))`  can be written as `is_array($qtyValue)` (2x)